showCompetences.js

```diff
@@ -0,0 +1,62 @@
+const users = [
+    {
+        userName: "Mario Rossi",
+        competenceUser: [
+            {
+                nameComp: 'java',
+                area: 'linguaggi programmazione',
+                level: 3
+            },
+            {
+                nameComp: 'Linux',
+                area: 'Sistemi operativi',
+                level: 2
+            }
+        ]
+    },
+    {
+        userName: "Franco Verdi",
+        competenceUser: [
+            {
+                nameComp: 'js',
+                area: 'linguaggi programmazione',
+                level: 4
+            },
+            {
+                nameComp: 'Windows',
+                area: 'Sistemi operativi',
+                level: 1
+            }
+        ]
+    }
+];
+
+for (const user of users) {
+    document.getElementById("show").innerHTML += `
+        <ul><br>
+            <li> Nome e Cognome: ` + user.userName + `</li><br>
```

Comment:
il backtick è ganzo proprio perché puoi usare le string template

```suggestion
            <li> Nome e Cognome: ${ user.userName } </li><br>
```